src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = ((a, b) => (
+    a.toString() > b.toString()
+  ))) {
```

Comment:
will be better to write this function using function declaration separately from function arguments
also, you need to write default sort functionality (function should return 0 if elements are the same) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort